HiringApp/HiringApp/Application/Scenes/ContactForm/ContactFormViewController.swift

```diff
@@ -0,0 +1,294 @@
+//
+//  ContactFormViewController.swift
+//  HiringApp
+//
+//  Created by Alba Luján on 28/6/17.
+//  Copyright (c) 2017 Visual Engineering. All rights reserved.
+//
+
+import UIKit
+
+enum InputTextType: Int {
+    case name = 0
+    case surname
+    case linkedin
+    case address
+    case phoneNumber
+}
+
+enum TextViewInputState {
+    case correct
+    case incorrect
+}
+
+private enum Constants {
+    
+    // Colors
+    static let brandBlue: UIColor =  UIColor(red: 101/255.0, green: 174/255.0, blue: 242/255.0, alpha: 1.0)
+    static let highlightedButtonColor: UIColor = UIColor(red: 1, green: 1, blue: 1, alpha: 0.5)
+    static let titleLabelTextColor: UIColor = .white
+    static let stackViewBackgroundColor: UIColor = .white
+    static let formLabelTextColor: UIColor = .black
+    static let wrongInputColor: UIColor = .red
+    static let correctInputColor: UIColor = .darkGray
+    
+    // Text
+    static let titleLabelText = R.string.localizable.cf_title()
```

Comment:
Why saving text resources in static properties? What happens when an application has a inner method to update its   displayed language?